src/Pages/Detail/Components/Utils.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import Styled from "styled-components";
+import { AiOutlineClose } from "react-icons/ai";
+
+export const API = "http://localhost:3000/Data/Detail.json";
+// export const API = "http://10.58.1.45:8000/product/1";
+export const APIPOST = "http://10.58.1.45:8000/product/1/post";
+
+export const Hr = ({ margin }) => {
+  return (
+    <HrWrap margin={margin}>
+      <hr />
+    </HrWrap>
+  );
+};
+
+export const Tooltip = () => {
+  const [istooltip, setIstooltip] = useState(false);
+
+  const handleToopTip = () => {
+    setIstooltip((prev) => !prev);
+  };
+  return (
+    <Wrap onClick={handleToopTip} istooltip={istooltip}>
+      <span>주소를 몰라도 간편하게!</span>
+      <div>
+        <AiOutlineClose />
+      </div>
+    </Wrap>
+  );
+};
+
+const Wrap = Styled.div`  
+display: flex;
+visibility:${(props) => props.istooltip && "hidden"};
+align-items: center;
+z-index: 10px;
+position: absolute;
+top: -50px;
+right: 0;
+background: ${(props) => (props.istooltip ? "white" : "black")};
```

Comment:
```suggestion
background: ${({ isToolTip }) => (isToolTip ? "white" : "black")};
```
- 구조분해 할당 가능합니다
- 카멜케이스로 작성해주세요